Tasks/Gradle/Gradle.ps1

```diff
@@ -76,6 +88,33 @@ if ($jdkPath)
     Write-Verbose "JAVA_HOME set to $env:JAVA_HOME"
 }
 
+$buildRootPath = Split-Path $wrapperScript -Parent
+$reportDirectoryName = "CodeCoverage"
+$reportDirectory = Join-Path $buildRootPath $reportDirectoryName
+
+if(Test-Path $reportDirectory)
+{
+   # delete any previous code coverage data 
+   rm -r $reportDirectory -force | Out-Null
```

Comment:
Its Ok not to delete this data in such cases its in use (again a very low chances - Since build would be executing by then)